src/errors/top_level.cr

```diff
@@ -3,7 +3,7 @@ def raise(error : Mint::Error.class)
   raise error, {} of String => Mint::Error::Value
 end
 
-def raise(error : Mint::Error.class, raw : Hash(String, T)) forall T
+def raise(error : Mint::Error.class, raw : Hash(String, Mint::Error::Value?))
```

Comment:
Why is this needed?

Reply:
It was just a compilation error change I was making along the way. Forgot to revert it.